javalin-testtools/src/main/java/io/javalin/testtools/SseEvent.kt

```diff
@@ -0,0 +1,8 @@
+package io.javalin.testtools
+
+import okhttp3.OkHttpClient
+import okhttp3.sse.EventSource
+
+data class SseEvent(val client: OkHttpClient, val source: EventSource, val id: String?, val type: String?, val data: String) {
+    fun closeClient() = source.cancel().also { client.dispatcher.executorService.shutdownNow() }
```

Comment:
Maybe this duplicated function could be moved into a single data class which has both client and eventsource?